### DISCUSSION TITLE
Change E-Mail Address in Profile

### DISCUSSION THREAD

**Discussion by MaPeras** [knowledgefutures/pubpub#2975] [Support]
Hi,

a rather trivial question but somehow important? :) How can I do that?

Cheers

Max

**Comment by meeloun** [comment]
When choosing biological ghostwriting        [https://www.lunwenhui.com/biologydaixie.html](https://www.lunwenhui.com/biologydaixie.html )       services, it is necessary to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, we can help you find personalized ghostwriting services that truly meet your needs.

**Comment by tefkah** [comment]
Hi, maybe better (extremely) late than never, but this is now possible if you go to https://pubpub.org/legal/settings !
